builtin/providers/azurerm/provider.go

```diff
@@ -124,6 +124,9 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_sql_database":      resourceArmSqlDatabase(),
 			"azurerm_sql_firewall_rule": resourceArmSqlFirewallRule(),
 			"azurerm_sql_server":        resourceArmSqlServer(),
+
+			"azurerm_web_app":     resourceArmWebApp(),
+			"azurerm_server_farm": resourceArmServerFarm(),
```

Comment:
Update the naming conventions. It should align with Microsoft product names now.